### DISCUSSION TITLE
Python Media player app created using customtkinter

### DISCUSSION THREAD

**Discussion by Mr-G254** [TomSchimansky/CustomTkinter#2199] [Showcase]
# **Media-Player-Desktop-App**
I have used python and customtkinter to create a media player app. This app plays video and audio providing an impressive UI. Apart from playing videos and audios you can also search for You Tube videos through this app and also download You Tube videos and audios. Here is the link to the project (Approximately 2.4k lines of code):
``` 
  https://github.com/Mr-G254/Media-Player-Desktop-App
  ```
## **App screenshots**
![playlist](https://github.com/TomSchimansky/CustomTkinter/assets/75112735/7d7d890e-4fee-420d-90ef-a43bd2d8ce02)

![songs](https://github.com/TomSchimansky/CustomTkinter/assets/75112735/9cd36767-1dfb-4295-ab5b-a9303591b27c)

![videos](https://github.com/TomSchimansky/CustomTkinter/assets/75112735/ccb628f9-c4cb-40a5-920e-789bc6c5ac0f)

![youtube](https://github.com/TomSchimansky/CustomTkinter/assets/75112735/f24e550a-703a-4b3a-8d46-5d659fbcb31a)

**Comment by SarthakTools** [comment]
Please see my modern spotify player with customtkinter...
https://github.com/SarthakTools/Spotify
